tests/CMakeLists.txt

```diff
@@ -40,11 +40,7 @@ else()
   set(SUCCESS_STATUS_OFFLOAD TRUE)
 endif()
 
-if(QMC_CUDA)
-  set(SUCCESS_STATUS_CUDA FALSE)
-else()
-  set(SUCCESS_STATUS_CUDA TRUE)
-endif()
+set(SUCCESS_STATUS_CUDA TRUE)
```

Comment:
This also needs to be removed.

Reply:
Thanks. SUCCESS_STATUS_CUDA is now completely removed.